docs/embedding.md

```diff
@@ -57,7 +57,6 @@ See it in action:
 | getStartedPayload | string | "GET_STARTED" | The payload to send to your Flow when clicking the Get Started Button / when sending the auto message. 
 | getStartedButtonText | string | "GET_STARTED" | The text to display on the Get Started Button / when sending the auto message. 
 | inputPlaceholder | string | "Write a reply" | The placeholder text to display in the input field. 
-| enableSTT | boolean | false | Whether to enable speech input that lets the user speak to the Webchat instead of only typing. 
```

Comment:
I think instead of removing this line we should instead add a short notice that the plugin needs to be loaded for this to work.